### PR TITLE
Modify router title

### PR DESCRIPTION
**Pull request by david8388**

### CODE REVIEW

**Comment by david8388** [comment]
Are you sure is Git Practice?

**Comment by DavidWus** [comment]
I'm not really sure... Haha

File: routes/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ var router = express.Router();
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+  res.render('index', { title: 'Git Practice' });
```
Comment (david8388) [inline routes/index.js:6]: Are you really sure it's Git Practice?

**Comment by david8388** [comment]
Ok. I merged.

**Pull request merged**